style.css

```diff
@@ -0,0 +1,151 @@
+
+@font-face {
+  font-family: Poppins;
+  src: url(fonts/Poppins-Regular.ttf);
+}
+
+@font-face {
+  font-family: Poppins-light;
+  src: url(fonts/Poppins-Light.ttf);
+}
+
+@font-face {
+  font-family: Poppins-extrabold;
+  src: url(fonts/Poppins-ExtraBold.ttf);
+}
+
+@font-face {
+  font-family: Poppins-semibold;
+  src: url(fonts/Poppins-SemiBold.ttf);
+}
+
+a {
+  text-decoration: none;
+}
+
+li {
+  list-style: none;
+}
+
+label {
+  display: none;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html,
+body {
+  margin: 0;
+  font-family: "Poppins", sans-serif;
+  line-height: 1.5em;
+  display: flex;
+  flex-direction: column;
+  color: #172b4d;
+  background-image: url("./header-shapes mobile.png");
+  background-repeat: no-repeat;
+  background-position: 0 8.0375em;
```

Comment:
**OPTIONAL** - Nice job making your website responsive, but to improve the UI of your site, I suggest making the background image responsive with respect to the viewport width(vw). Please fix this 🙏 

Your project:
![tnsdzdfbz](https://user-images.githubusercontent.com/73607512/227522826-8c1bb7cf-fad2-4fc9-b60f-a3c10d3a1a3a.PNG)